Dockerfile-7.2

```diff
@@ -48,7 +47,7 @@ RUN phpdismod xdebug
 # Add the blackfire repo and install the php-probe.
 # We also fetch the blackfire-agent to get access to the commandlineuploader.
 RUN \
-  wget -O - https://packagecloud.io/gpg.key | apt-key add - && \
+  wget -O - https://packages.blackfire.io/gpg.key | apt-key add - && \
```

Comment:
Huh? Hopefully right.

Reply:
According to https://blackfire.io/docs/up-and-running/installation it is. Also the old one didn't work anymore.